<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Andrey Rijov</author>
 <class>Add_New_Device_Window</class>
 <widget class="QDialog" name="Add_New_Device_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>299</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Storage Device</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/blockdevice.png</normaloff>:/blockdevice.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="5">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item>
      <widget class="QCheckBox" name="CH_File">
       <property name="text">
        <string>&amp;File</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Edit_File_Path">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="TB_File_Path_Browse">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/open-file.png</normaloff>:/open-file.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QCheckBox" name="CH_Interface">
     <property name="text">
      <string>&amp;Interface</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="3">
    <widget class="QComboBox" name="CB_Interface">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>ide</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>scsi</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>sd</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>mtd</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>floppy</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>pflash</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>virtio</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QCheckBox" name="CH_Index">
     <property name="text">
      <string>Index (For IDE, Floppy)</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QSpinBox" name="SB_Index">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QCheckBox" name="CH_Bus_Unit">
     <property name="text">
      <string>Bus, Unit (For SCSI)</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QSpinBox" name="SB_Bus">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QSpinBox" name="SB_Unit">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QCheckBox" name="CH_Media">
     <property name="text">
      <string>&amp;Media</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2" colspan="3">
    <widget class="QComboBox" name="CB_Media">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>Disk</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>CD-ROM</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QCheckBox" name="CH_Cache">
     <property name="text">
      <string>Cac&amp;he</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2" colspan="3">
    <widget class="QComboBox" name="CB_Cache">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>none</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>writeback</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>writethrough</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QCheckBox" name="CH_AIO">
     <property name="text">
      <string>AIO</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2" colspan="3">
    <widget class="QComboBox" name="CB_AIO">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>threads</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>native</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QCheckBox" name="CH_Snapshot">
     <property name="text">
      <string>&amp;Snapshot</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2" colspan="3">
    <widget class="QComboBox" name="CB_Snapshot">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>on</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>off</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QCheckBox" name="CH_Boot">
     <property name="text">
      <string>Boot</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2" colspan="3">
    <widget class="QComboBox" name="CB_Boot">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>on</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>off</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="9" column="1" colspan="4">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>200</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="10" column="0" colspan="5">
    <widget class="QGroupBox" name="GB_hdachs_Settings">
     <property name="title">
      <string>Settings for -hdachs Option</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <property name="margin">
       <number>6</number>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="0" column="1">
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2">
       <layout class="QGridLayout">
        <property name="margin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>6</number>
        </property>
        <item row="1" column="1">
         <widget class="QLineEdit" name="Edit_Trans"/>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="Edit_Secs"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="Label_Secs">
          <property name="text">
           <string>s&amp;ecs</string>
          </property>
          <property name="buddy">
           <cstring>Edit_Secs</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="Label_Trans">
          <property name="text">
           <string>&amp;trans</string>
          </property>
          <property name="buddy">
           <cstring>Edit_Trans</cstring>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <layout class="QGridLayout">
        <property name="margin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>6</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="Label_Cyls">
          <property name="text">
           <string>c&amp;yls</string>
          </property>
          <property name="buddy">
           <cstring>Edit_Cyls</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="Edit_Cyls"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="Label_Heads">
          <property name="text">
           <string>&amp;heads</string>
          </property>
          <property name="buddy">
           <cstring>Edit_Heads</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="Edit_Heads"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="11" column="0" colspan="5">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>120</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="Button_OK">
       <property name="text">
        <string>&amp;OK</string>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/apply.png</normaloff>:/apply.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Button_Cancel">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/cancel.png</normaloff>:/cancel.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>CH_File</tabstop>
  <tabstop>Edit_File_Path</tabstop>
  <tabstop>TB_File_Path_Browse</tabstop>
  <tabstop>CH_Interface</tabstop>
  <tabstop>CB_Interface</tabstop>
  <tabstop>CH_Index</tabstop>
  <tabstop>SB_Index</tabstop>
  <tabstop>CH_Bus_Unit</tabstop>
  <tabstop>SB_Bus</tabstop>
  <tabstop>SB_Unit</tabstop>
  <tabstop>CH_Media</tabstop>
  <tabstop>CB_Media</tabstop>
  <tabstop>CH_Cache</tabstop>
  <tabstop>CB_Cache</tabstop>
  <tabstop>CH_AIO</tabstop>
  <tabstop>CB_AIO</tabstop>
  <tabstop>CH_Snapshot</tabstop>
  <tabstop>CB_Snapshot</tabstop>
  <tabstop>CH_Boot</tabstop>
  <tabstop>CB_Boot</tabstop>
  <tabstop>GB_hdachs_Settings</tabstop>
  <tabstop>Edit_Cyls</tabstop>
  <tabstop>Edit_Secs</tabstop>
  <tabstop>Edit_Heads</tabstop>
  <tabstop>Edit_Trans</tabstop>
  <tabstop>Button_OK</tabstop>
  <tabstop>Button_Cancel</tabstop>
 </tabstops>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>Button_Cancel</sender>
   <signal>clicked()</signal>
   <receiver>Add_New_Device_Window</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>401</y>
    </hint>
    <hint type="destinationlabel">
     <x>314</x>
     <y>331</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CH_Interface</sender>
   <signal>toggled(bool)</signal>
   <receiver>CB_Interface</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>168</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CH_Media</sender>
   <signal>toggled(bool)</signal>
   <receiver>CB_Media</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>177</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CH_File</sender>
   <signal>toggled(bool)</signal>
   <receiver>Edit_File_Path</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>44</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>30</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CH_File</sender>
   <signal>toggled(bool)</signal>
   <receiver>TB_File_Path_Browse</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>303</x>
     <y>34</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CH_Index</sender>
   <signal>toggled(bool)</signal>
   <receiver>SB_Index</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>106</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CH_Bus_Unit</sender>
   <signal>toggled(bool)</signal>
   <receiver>SB_Bus</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>104</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CH_Bus_Unit</sender>
   <signal>toggled(bool)</signal>
   <receiver>SB_Unit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>293</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CH_Cache</sender>
   <signal>toggled(bool)</signal>
   <receiver>CB_Cache</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>51</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CH_AIO</sender>
   <signal>toggled(bool)</signal>
   <receiver>CB_AIO</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>33</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>120</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CH_Snapshot</sender>
   <signal>toggled(bool)</signal>
   <receiver>CB_Snapshot</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>226</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CH_Boot</sender>
   <signal>toggled(bool)</signal>
   <receiver>CB_Boot</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>51</x>
     <y>261</y>
    </hint>
    <hint type="destinationlabel">
     <x>167</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
